sql/expression/logic.go

```diff
@@ -129,6 +137,30 @@ func NewOr(left, right sql.Expression) sql.Expression {
 	return &Or{BinaryExpression{Left: left, Right: right}}
 }
 
+// JoinOr joins several expressions with Or.
+func JoinOr(exprs ...sql.Expression) sql.Expression {
```

Comment:
Would you be willing to update the unit tests in `logic_test.go` to add some tests for this function? There are some existing tests there for `JoinAnd` already that would be a good template.